src/components/Tooltip/Tooltip.tsx

```diff
@@ -83,6 +86,9 @@ export const Tooltip = ({
   text,
   ...rest
 }: TooltipProps) => {
+  if (variant === 'dark' && process.env.NODE_ENV !== 'production') {
+    console.warn('Dark variant is deprecated.');
+  }
```

Comment:
Maybe we could add a little more explanation, like "The dark variant is deprecated and will be removed in an upcoming release. Please use the light variant instead."? I'm afraid people might not know what "deprecated" means in this context because this is the first time we're doing it.

Reply:
Sounds good, thanks for the suggestion, will change